src/studio-home/tabs-section/courses-tab/courses-filters/courses-filter-menu/index.jsx

```diff
@@ -26,6 +27,13 @@ const CoursesFilterMenu = ({
     if (cleanFilters) {
       setItemMenuSelected(defaultItemSelectedText);
```

Comment:
@pkulkark Don't you just need to call the `onItemMenuSelected` here? Or even better:
```suggestion
    const defaultItemSelectedValue = menuItems.find(item => item.name === defaultItemSelectedText);
    handleCourseTypeSelected(defaultItemSelectedText, defaultItemSelectedValue);
```

Reply:
@Cup0fCoffee I actually couldn't fully figure out how the `cleanFilters` option could be updated, so I didn't change the existing behaviour.